examples/apps/cli/main.c

```diff
@@ -93,6 +93,11 @@ static const otCliCommand kCommands[] = {
 };
 #endif // OPENTHREAD_POSIX && !defined(FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION)
 
+#if OPENTHREAD_CONFIG_CLI_VENDOR_COMMANDS_ENABLE
+extern const otCliCommand gVendorCommands[];
```

Comment:
Is it possible to call `otCliSetUserCommands()` in vendor's code?

Reply:
@bukepo - thanks so much for the review.  I'll take a look now.

Reply:
@bukepo what are your thoughts on replacing this code:

```
#if OPENTHREAD_CONFIG_CLI_VENDOR_COMMANDS_ENABLE && OPENTHREAD_CONFIG_CLI_MAX_USER_CMD_ENTRIES > 1
    IgnoreError(otCliSetUserCommands(gVendorCommands, gVendorCommandsLength, instance));
#endif

```
with code to call out to vendor's code.

```
#if OPENTHREAD_CONFIG_CLI_VENDOR_COMMANDS_ENABLE
    IgnoreError(otCliVendorSetUserCommands(instance));
#endif

```

Reply:
@bukepo I just pushed the above suggested change.  Please have another look.  Thank you.